docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -13,7 +13,7 @@ The following parameters define applications, individuals, teams, verticals or a
 * Stakeholder groups
 * Job functions
 * Business services
-* Tag types
+* Categories
```

Comment:
can you please replace with **Tag categories** ?

Reply:
Done